### PR TITLE
IN: Profile Page > Edit Profile

### PR DESCRIPTION
**Pull request by Braagaa**
**What it does:**
Allows User to update profile settings which causes changes in the database to persist data.

**What to look for:**

It might still be buggy
Email does not currently reflect changes because the user object context does not get updated
Maybe put Email in profile to reflect changes (double update on User and Profile)

### CODE REVIEW

**Comment by dproc96** [comment]
Can you merge in the changes from dev? You've changed the architecture of the Profile Settings page in here, I think the routing I added to dev must've been merged after you created this branch. This is the structure I used when making it.
```
<Card className={classes.card}>
          <Route path={`${origin}/edit-profile`} />
          <Route path={`${origin}/profile-photo`} component={EditProfilePhoto} />
          <Route path={`${origin}/payment`} />
          <Route path={`${origin}/security`} />
          <Route path={`${origin}/settings`} />
</Card>
```

**Comment by Braagaa** [comment]
Okay I just merged it!

File: client/src/App.js [hunk 1]
```diff
@@ -2,32 +2,36 @@ import React from "react";
...
-          <Route exact path="/" component={LandingPage} />
+          <Route exact path="/"/>
           <Route exact path="/signup" component={SignUp} />
           <Route path="/login" component={Login} />
           <PrivateRoute path="/me" component={ProfileSettings} />
-          <PrivateRoute exact path="/become-a-sitter" component={LandingPage} />
-          <PrivateRoute exact path="/my-sitters" component={LandingPage} />
-          <PrivateRoute exact path="/my-jobs" component={LandingPage} />
-          <PrivateRoute exact path="/messages" component={LandingPage} />
+		  <PrivateRoute path="/me/profile-edit">
```
Comment (dproc96) [inline client/src/App.js]: Can you remove this portion as the merge from dev covers this?

**Review by dproc96 — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (Braagaa) [reply client/src/App.js]: check it out @dproc96

**Review by dproc96 — APPROVED** [review]

**Pull request merged**